src/data/battle_partners.h

```diff
@@ -0,0 +1,32 @@
+const struct Trainer gBattlePartners[] = {
+    [TRAINER_NONE] =
+    {
+        .trainerClass = TRAINER_CLASS_PKMN_TRAINER_1,
+        .encounterMusic_gender = TRAINER_ENCOUNTER_MUSIC_MALE,
+        .trainerPic = TRAINER_PIC_HIKER,
+        .trainerName = _(""),
+        .items = {},
+        .doubleBattle = FALSE,
+        .aiFlags = 0,
+        .partySize = 0,
+        .party = NULL,
```

Comment:
I'd expect this dummy/reference/whatever section to mirror the layout of the actual existing partners.